react-main/src/redux/profile/reducer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { actions } from './actions';
+
+const initialState = { info: '' };
+
+export function reducerScore(state = initialState, action) {
+  switch (action.type) {
```

Comment:
If there's no change, you don't need to define the case. If an action is triggered but no one is listening, it won't fail. Remove the case (and the reducer, as it's the only casi it has)